tests/mender-conductor-enterprise/server/workflows/create_organization.json

```diff
@@ -0,0 +1,152 @@
+{
```

Comment:
Where are these workflows files coming from?

I don't think that any of these workflows files belong here in the first place, and on top of that I am very skeptical into committing "enterprise" files into an OS repository.

Reply:
Also, I cannot review the content, someone else should do that.